src/astra_driver.cpp

```diff
@@ -729,6 +729,15 @@ sensor_msgs::msg::CameraInfo::SharedPtr AstraDriver::getDepthCameraInfo(int widt
 void AstraDriver::readConfigFromParameterServer()
 {
   depth_frame_id_ = std::string("openni_depth_optical_frame");
+
+  // Load the depth registration parameter, which may have been set before
+  //   driver initialization.
+  pnh_->get_parameter("depth_registration", depth_registration_);
+  if(depth_registration_)
+  {
+    ROS_INFO("Astra depth registration enabled");
+  }
```

Comment:
Ug, sorry about this.  I also realized that it might be better to always print, and just print `enabled` or `disabled` as appropriate.  Something like:

```
ROS_INFO("Astra depth registration %s", depth_registration_ ? "enabled" : "disabled");
```